tests/pytorch/mock_sparse/test_elementwise_op_sp.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pytest
+import dgl
+import dgl.backend as F
+import torch
+import numpy
+import operator
+from dgl.mock_sparse import elementwise_op_sp
+from dgl.mock_sparse.sp_matrix import SparseMatrix
+parametrize_idtype = pytest.mark.parametrize("idtype", [F.int32, F.int64])
+parametrize_dtype = pytest.mark.parametrize('dtype', [F.float32, F.float64])
+
+def all_close_sparse(A, B):
+    assert torch.allclose(A.indices(), B.indices())
+    assert torch.allclose(A.values(), B.values())
+    assert A.shape == B.shape
+
+@parametrize_idtype
+@parametrize_dtype
+@pytest.mark.parametrize('op', [operator.add, operator.sub, operator.mul, operator.truediv])
+def test_sparse_op_sparse(idtype, dtype, op):
```

Comment:
Please add a similar test for `dgl.mock_sparse.add(A, B)`, etc. This is to verify that the APIs are correctly exposed.